File: model/location.go
package model

type Location struct {
	ID      int     `json:"id"`
	Display string  `json:"display"`
	Matcher string  `json:"matcher"`
	Lat     float32 `json:"lat"`
	Long    float32 `json:"long"`
	Zone    string  `json:"zone"`
	Site    string  `json:"site"`
}

File: src/Wanderinglunch.Core/Entities/Location.cs
namespace Wanderinglunch.Core.Entities
{
    public class Location
    {
        public long Id { get; set; }
        public string Display { get; set; }
        public string Matcher { get; set; }
        public float Lat { get; set; }
        public float Long { get; set; }
        public string Zone { get; set; }
        public string Site { get; set; }
    }
}